LANDSCAPE/COMPARISONS/ARGOCD-FLUX/index.md

```diff
@@ -0,0 +1,275 @@
+# Continuous Deployment on Kubernetes: What’s the Difference Between Argo CD and FluxCD
+
+If you’ve heard of GitOps and its benefits, you may have come across two open source projects in the Cloud Native Computing Foundation: Argo CD and FluxCD. Members of these projects are involved in the CNCF’s OpenGitOps initiative, and both are incubating projects. With GitOps, you increase reliability and velocity, lower downtime, and can enjoy other benefits through key GitOps capabilities such as reconciliation.
+
+When building out a Kubernetes cluster, the actual pipelines for the code behind applications is just as important as the firewalls, load balancers and provisioning systems at the front. There are many ways to build out application CI/CD pipelines in Kubernetes, but in this article, we are going to focus very specifically on two options for Continuous Deployment: [Flux CD](https://fluxcd.io/) (specifically Flux v2) and [Argo CD](https://argoproj.github.io/cd/).
```

Comment:
CI/CD → CI & CD
```suggestion
When building out a Kubernetes cluster, the actual pipelines for the code behind applications is just as important as the firewalls, load balancers and provisioning systems at the front. There are many ways to build out application CI & CD pipelines in Kubernetes, but in this article, we are going to focus very specifically on two options for Continuous Deployment: [Flux CD](https://fluxcd.io/) (specifically Flux v2) and [Argo CD](https://argoproj.github.io/cd/).
```

Reply:
The most common reference is CI/CD not CI & CD.

Reply:
@chris-short Understood.

How about removing the phrase "CI/CD"?

application CI/CD pipelines → application pipelines

```suggestion
When building out a Kubernetes cluster, the actual pipelines for the code behind applications is just as important as the firewalls, load balancers and provisioning systems at the front. There are many ways to build out application pipelines in Kubernetes, but in this article, we are going to focus very specifically on two options for Continuous Deployment: [Flux CD](https://fluxcd.io/) (specifically Flux v2) and [Argo CD](https://argoproj.github.io/cd/).
```
‎
For reference: [AWS Container Day: Kubernetes Edition KubeCon 2020](https://www.youtube.com/watch?v=o2_sGFkYNvA&t=1184s) by @countspongebob
> It is really critical to not conflate CI and CD. I try hard to not use the phrase CI/CD because they are very different practices. One of the lovely things about the GitOps model is the clean and understandable relationship that is created between CI as a Dev practice, and CD as an Ops practice. I think the notion of the Immutability Firewall in between them based on Git is really excellent.